inc/projecttasktemplate.class.php

```diff
@@ -199,30 +199,35 @@ function rawSearchOptions() {
       return $tab;
    }
 
-
-   function displaySpecificTypeField($ID, $field = []) {
-
+   public function getSpecificTypeField(int $ID, array $field): string {
       switch ($field['type']) {
          case 'percent_done' :
-            Dropdown::showNumber("percent_done", ['value' => $this->fields['percent_done'],
-                                                  'min'   => 0,
-                                                  'max'   => 100,
-                                                  'step'  => 5,
-                                                  'unit'  => '%']);
-            break;
+            return Dropdown::showNumber("percent_done", [
```

Comment:
do this change from displaying html to return it will cause some break in other glpi displays ?

Reply:
It is possible for some plugin to have issues with this if they are using it for their own forms, but I don't see any instances of other GLPI core code calling this method or in the plugins I have in my workspace.

Reply:
ok for me